doc/source/user_guide/data_sources_tutorial.rst

```diff
@@ -0,0 +1,310 @@
+.. _user_guide_data_sources:
+
+===============
+DPF DataSources
+===============
+
+The DataSources operator is a class available as a submodule of the 'ansys.dpf.core'
+package. It contains files with the analysis results. So this object is used in
+the first basic step: define simulation data, by defining where it is located.
+
+This example will describe the functionalities and functions of the class DataSources
+and can be generalised for all the files formats accepted by Py-DPF:
+
+.. _list_extensions_solvers:
+**Supported apps by DPF and their related formats**
+
+
+   - MAPDL files: .rst, .mode, .rfrq, .rdsp
```

Comment:
Also, maybe what we will want to link to/include another .rst file with this list, as it is best to not duplicate this kind of info. Imagine we support new formats, then we have to remember to update this list everywhere.

Reply:
Yes I see. I'll do that list in a separate file